portfolio/src/main/webapp/index.html

```diff
@@ -59,6 +59,16 @@ <h3> Comment </h3>
           <br><br>
           <input type="submit" />
        </form>
+       <br>
+       <label for="num-of-comments"> Comments Displayed : </label>
+       <select name="comment-limit-selector" id="comment-limit-selector" onChange="updateCommentLimit()">
+         <option value="5">5</option>
+         <option selected value="10">10</option>
+         <option value="20">20</option>
+         <option value="50">50</option>
+         <option value="All">All</option>
```

Comment:
A dropdown was a good move, since it prevents people from entering weird values.